etc/kayobe/environments/ci-aio/automated-setup.sh

```diff
@@ -88,7 +96,22 @@ kayobe overcloud host configure
 
 kayobe overcloud service deploy
 
-export KAYOBE_CONFIG_SOURCE_PATH=$BASE_PATH/src/kayobe-config
-export KAYOBE_VENV_PATH=$BASE_PATH/venvs/kayobe
-pushd $BASE_PATH/src/kayobe
-./dev/overcloud-test-vm.sh
+if $AIO_RUN_TEMPEST; then
+    pushd $BASE_PATH/src/kayobe-config
+    git submodule init
+    git submodule update
+    sudo DOCKER_BUILDKIT=1 docker build --build-arg BASE_IMAGE=rockylinux:9 --file .automation/docker/kayobe/Dockerfile --tag kayobe:latest --network host .
+    export KAYOBE_AUTOMATION_SSH_PRIVATE_KEY=$(cat ~/.ssh/id_rsa)
+    mkdir -p tempest-artifacts
+    sudo -E docker run --name kayobe-automation --detach -it --rm --network host \
+    -v $(pwd):/stack/kayobe-automation-env/src/kayobe-config -v $(pwd)/tempest-artifacts:/stack/tempest-artifacts \
+    -e KAYOBE_ENVIRONMENT -e KAYOBE_VAULT_PASSWORD -e KAYOBE_AUTOMATION_SSH_PRIVATE_KEY kayobe:latest \
+    /stack/kayobe-automation-env/src/kayobe-config/.automation/pipeline/tempest.sh -e ansible_user=stack
```

Comment:
I had wondered if you could just do a `kayobe playbook run .automation/ansible/<tempest>.yml`. Might be worth a try if you still have an aio.

Reply:
Still a little barebones for now but I've written up a playbook to run Tempest. Needs some refinement (e.g. some validation, more steps automated, better docs) but we're already well outside the scope of the PR I started with and a million miles away from the firewall config I was meant to be working on.

Reply:
The .automation playbook expects a fair bit of environment-variable-based setup. It's doable but not in 5 minutes

Reply:
And yes, I have now seen that you put `.automation/ansible` not `etc/kayobe/ansible` and the whole thing makes way more sense

Reply:
Reverted